Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -446,6 +446,59 @@
         - ReagentId: Fiber
           Quantity: 10
 
+- type: entity
+  name: cotton dough slice
+  parent: FoodBakingBase
+  id: FoodDoughCottonSlice
+  description: A slice of cotton dough.
+  components:
+  - type: FlavorProfile
+    flavors:
+    - dough
+    - cotton
+  - type: Sprite
+    state: cotton-dough-slice
+  - type: Tag
+    tags:
+    - Slice
+  - type: Construction
+    graph: DoughRopeCotton
+    node: start
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 18
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 7.5
+        - ReagentId: Fiber
+          Quantity: 7.5
```

Comment:
why do the 3 slices have more reagents than the whole dough had before?

Reply:
All regular dough slices and dough ropes inherit FoodBakingBase which has 15 nutriment, which means that you can turn 15 nutriment dough into 3 dough ropes/slices worth 45 nutriment combined.

Slam requested that I halve the nutriment content and replace it with fiber so that the moths actually get food out of the food.

Do you want me to fix the normal and moff dough solution contents so they actually contain the correct amount for 3 parts?

Reply:
yeah, fix the normal one as well please